README.rst

```diff
@@ -5,7 +5,7 @@ desitemplate
 Introduction
 ============
 
-This repository is intended to be a template for other repositories.
+This repository is intended to be a template for other **Python** repositories.
```

Comment:
Perhaps mention that this is specific to DESI with a link to desi.lbl.gov, since this is the first thing some sees when they visit this repo on github.